src/Stepper/VerticalStep.jsx

```diff
@@ -0,0 +1,338 @@
+import React, {PropTypes} from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+
+import {getMuiTheme} from '../styles';
```

Comment:
this one too. always directly import the modules you need. helps trim the bundle down a bit.